charts/presto/templates/deployment-worker.yaml

```diff
@@ -39,8 +39,10 @@ spec:
         checksum/catalog: {{ include (print $.Template.BasePath "/configmap-catalog.yaml") . | sha256sum }}
         checksum/worker: {{ include (print $.Template.BasePath "/configmap-worker.yaml") . | sha256sum }}
     spec:
-      {{- with .Values.imagePullSecrets }}
-      imagePullSecrets: {{- toYaml . | nindent 8 }}
+      {{- if .Values.velox.enabled }}
+      imagePullSecrets: {{- toYaml .Values.velox.imagePullSecrets | nindent 8 }}
+      {{- else }}
+      imagePullSecrets: {{- toYaml .Values.imagePullSecrets | nindent 8 }}
```

Comment:
Could you please revert it, because there is no `{{ .Values.velox.imagePullSecrets }}` anymore?